src/meta/app/src/principal/user_grant.rs

```diff
@@ -21,19 +21,31 @@ use enumflags2::BitFlags;
 use crate::principal::UserPrivilegeSet;
 use crate::principal::UserPrivilegeType;
 
-/// [`GrantObjectByID`] is used to maintain the grant object by id. Using ID over name
+/// [`OwnerObject`] is used to maintain the grant object that support rename by id. Using ID over name
```

Comment:
OwnerObject -> OwnershipObject ?

OwnerObject might have a bit of confuse about it represents the `owner`, but `OwnerObject` not targets an owner, but the object of an ownership.

Reply:
Done